cloudferrylib/os/actions/cinder_database_manipulation.py

```diff
@@ -109,6 +111,75 @@ def _clean_data(data):
     return data
 
 
+class CinderStorageMigrationProgressView(object):
```

Comment:
Minor: might be better to move this class to a separate file, migration progress does not seem to be a part of  `cinder_database_manipulation` module